src/NuGet.Core/NuGet.ProjectModel/Strings.resx

```diff
@@ -152,4 +152,58 @@
   <data name="MissingVersionOnTool" xml:space="preserve">
     <value>Tools must specify a version range.</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="PackagesLockFile_CentralPackageVersionRemoved" xml:space="preserve">
+    <value>Central package management file doesn't contain version range for {0} package which is specified as CentralTransitive dependency in the lock file.</value>
+  </data>
+  <data name="PackagesLockFile_IncompatibleLockFileVersion" xml:space="preserve">
+    <value>Version specified in the packages.lock.json file is invalid. Current tooling supports version {0} and previous formats.</value>
+  </data>
+  <data name="PackagesLockFile_MismatchedTargetFrameworks" xml:space="preserve">
+    <value>The project target frameworks are different than the lock file's target frameworks. Lock file target frameworks: {0}. Project target frameworks {1}.</value>
+    <comment>0 - Semicolon delimited list of lock file target frameworks, 1 - semicolon delimited list of lock file target frameworks</comment>
+  </data>
+  <data name="PackagesLockFile_NewTargetFramework" xml:space="preserve">
+    <value>The project target framework {0} was not found in the lock file.</value>
+    <comment>0 - project target framework</comment>
+  </data>
+  <data name="PackagesLockFile_PackageReferenceAdded" xml:space="preserve">
+    <value>A new package reference was found {0} for the project target framework {1}.</value>
+  </data>
+  <data name="PackagesLockFile_PackageReferencesHaveChanged" xml:space="preserve">
+    <value>The package references have changed for {0}. Lock file's package references: {1}, project's package references: {2}.</value>
+    <comment>0 - tfm, 1 - lock file package references, 2 - project package references</comment>
+  </data>
+  <data name="PackagesLockFile_PackageReferenceVersionChanged" xml:space="preserve">
+    <value>The package reference {0} version has changed from {1} to {2}.</value>
+  </data>
+  <data name="PackagesLockFile_ProjectIsMissingRestoreMetadataTfms" xml:space="preserve">
+    <value>Project is mising the required target framework {0}. Target frameworks are out of sync between Project and Restore metadata.Please file a bug with repro steps on NuGet/Home if seen.</value>
```

Comment:
nit: I wouldn't necessarily elaborate on that, rather than just ask for a feedback ticket. 
The customer doesn't care too much.

Reply:
Changed the message to `Please file a bug with repro steps on NuGet/Home or VS Feedback ticket if seen.`